src/Core/BitwardenIcons.cs

```diff
@@ -0,0 +1,118 @@
+namespace Bit.Core
+{
+    public static class BitwardenIcons
+    {
+        // TODO: Missing icons
+        public const string File = FilePdf;
+        public const string FolderOpenO = Frown;
+        public const string ChevronUp = Frown;
+        public const string Paste = Frown;
+        public const string Ban = Frown;
+        public const string Camera = Frown;
+        // Eye-slash not aligned with eye
+
+        public const string User = "";
```

Comment:
We've been transitioning towards using the unicode values for these instead of the actual chars since they aren't readable everywhere (like here for example).